a3p-integration/README.md

```diff
@@ -1,38 +1,107 @@
-# Integration with agoric-3 synthetic test chain
+# Overview
 
-The test runner is `@agoric/synthetic-chain`. This package depends on that so that you can run,
+This folder contains an end-to-end integration test executed against a synthetic agoric-3 chain. The test performs a chain software upgrade to the software contained in the enclosing `agoric-sdk` repository, then executes a series of functional tests verifying the upgrade accomplished its goal.
+
+# How to run
+
+## Pre-requisites
+
+The synthetic chain testing infrastructure relies on Docker and the Docker Buildx extended build capabilities. Make sure you have a recent Docker engine installed for your system.
+
+### Install test dependencies
+
+This directory hierarchy, while it contains packages, is not part of the `agoric-sdk` root Yarn project. This is to isolate it from tooling that expects a public package published to NPM. For example, it doesn't run in the CI jobs for the `/packages` packages.
+
+The end-to-end integration test relies on the https://www.npmjs.com/package/[@agoric/synthetic-chain](https://github.com/agoric/synthetic-chain) test tool (published to NPM from the [`agoric-3-proposals` repository](https://github.com/Agoric/agoric-3-proposals)), and uses Yarn 4 (configured through corepack).
```

Comment:
Not a valid link: https://github.com/agoric/synthetic-chain

```suggestion
The end-to-end integration test relies on the [@agoric/synthetic-chain](https://www.npmjs.com/package/@agoric/synthetic-chain) test tool (published to NPM from the [`agoric-3-proposals` repository](https://github.com/Agoric/agoric-3-proposals)), and uses Yarn 4 (configured through corepack).
```

Reply:
Ah I didn't check the previous commit that Ivan added!